package.json

```diff
@@ -13,6 +13,12 @@
     "coverage": "nyc --reporter=html npm run unit-tests",
     "system-tests": "./node_modules/mocha/bin/mocha test/system/**.test.js --timeout 5000"
   },
+  "com_postman_plugin": {
+    "plugin_type": "importer",
+    "name": "HAR",
+    "source_format": "HAR",
+    "source_format_name": "HAR"
```

Comment:
@diyaeninan We have `sample_input` here as well in other importers. We can include it as well.

Reply:
Sure, I can add this. It wasn't present in the curl module hence I thought it was not a required field